apps/pipelines/models.py

```diff
@@ -238,6 +238,28 @@ class Node(BaseModel, VersionsMixin):
     def __str__(self):
         return self.flow_id
 
+    def create_new_version(self, save: bool = True):
+        """
+        Create a new version of the node and if the node is an assistant node, create a new version of the assistant
+        and update the `assistant_id` in the node params to the new assistant version id.
+        """
+        from apps.assistants.models import OpenAiAssistant
+        from apps.pipelines.nodes.nodes import AssistantNode
+
+        assistant_node_name = AssistantNode.model_json_schema()["title"]
```

Comment:
Isn't this just `AssistantNode.__name__`

Reply:
Ah yes it is. Forgot about this fella, thanks